packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -293,7 +293,7 @@ export class List extends BaseComponent<IListProps, IListState> {
     let divProps = getNativeProps(this.props, divProperties);
 
     // assign list if no role
-    role = role || 'list';
+    role = (typeof role === 'undefined') ? 'list' : role;
```

Comment:
no, for a `typeof` check, `'undefined'` is correct.
(see: 
![image](https://cloud.githubusercontent.com/assets/2201819/24383746/bc03108a-1313-11e7-8fd8-ece1a4828e74.png))